modules/installation-custom-aws-vpc.adoc

```diff
@@ -44,6 +48,18 @@ endpoints should be named as follows:
 * `ec2.<region>.amazonaws.com`
 * `elasticloadbalancing.<region>.amazonaws.com`
 * `s3.<region>.amazonaws.com`
+endif::aws-china[]
+
+ifdef::aws-china[]
+If you are working in a disconnected environment, you are unable to reach the
```

Comment:
I would remove the limited spaces at the end of each of these lines.

Reply:
Thank you. Updated.